lib/cocoapods/downloader/cache.rb

```diff
@@ -197,7 +197,7 @@ def path_for_pod(request, slug_opts = {})
       #
       def path_for_spec(request, slug_opts = {})
         path = root + 'Specs' + request.slug(**slug_opts)
-        path.sub_ext('.podspec.json')
+        Pathname.new(path.to_path + '.podspec.json')
```

Comment:
does this change invalidate existing caches for folks?

Reply:
yes, since the cache path change, the old cache won't hit, and will keep until clear directory.